Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Troll Religious Revamp

Closes #1212 


## Changelog:
- All orthodox troll Loa religions start with Zul as head of faith
- Orthodox troll Loa religions gain the option to separate from Zandalari religious control, potentially causing a schism
  - Rulers of your faith will either side with you and accept the new head of faith, or side with Zul and convert to Zan'atali
  - The likelyhood of either option depends on their opinion of both you and Zul


## Developer changelog:
- All orthodox troll Loa religions specifically refers gurubism, drakkism, amism, farrism


## Tests:


# How to test:
 - Testing from other perspectives can be done by commenting out `every_player` from `wc_religious_decisions.txt` and removing the comments from the `days =` and `every_ruler` lines. After that activate the decision like normal and switch to the character whose perspective you'd like to test, before unpausing the game.
- Check that:
  - All localization looks correct
  - Head of Faith is replaced as intended
  - Rulers who disagree with the change convert as intended
- Test decision and events from perspectives
  - From the separator (the one who initiates the decision)
  - From Zul
  - From other Loa ruler of different faith
  - From neighboring ruler of the separator
